### ISSUE TITLE
Multi-line regex bodies removed

### ISSUE THREAD

**Issue by royaldark** [weavejester/cljfmt#76]
A multi-line regex with embedded comments loses its body when cljfmt runs. See the following contrived example:

```clojure
(def pattern
  #"(?x)
    ^   # Beginning of line
    .*  # Any number of characters
    $   # End of line")
```
`lein cljfmt fix` outputs:

```clojure
(def pattern
  #"(?x)")
```

Lein 2.6.1
cljfmt 0.5.3

**Comment by weavejester** [comment]
Thanks for the report. This is caused by a bug in rewrite-clj, and I've opened an issue for it.

**Comment by trimtab613** [comment]
The bug in rewrite-clj has been fixed: https://github.com/xsc/rewrite-clj/issues/50

**Comment by weavejester** [comment]
This is now fixed in cljfmt for Clojure (via #80), but not ClojureScript.

**Comment by trimtab613** [comment]
Awesome thank you. Looking forward to 0.5.4 being published on clojars!
